flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -149,20 +138,27 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
-				"mvn dependency:tree" as of Calcite 1.21:
+				"mvn dependency:tree" as of Calcite 1.22.0:
 
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.21.0:compile
-				[INFO] |  +- commons-codec:commons-codec:jar:1.10:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.21.0:compile
-				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.10.1:compile
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.22.0:compile
+				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.22.0:compile
 				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.1:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.1:compile
-				[INFO] |  +- com.google.guava:guava:jar:19.0:compile
```

Comment:
We should keep guava in the dependency tree, beacuse it is used in Calcite core and we will shade them into another package path.

Reply:
It looks like a big change on the dependency tree. cc @twalthr who has more experience on the dependencies.

Reply:
In Calcite 1.22, there is no guava in the calcite-core dependency, and we already have a explicit dependency.